gno.land/cmd/gnoland/testdata/getting_started.txtar

```diff
@@ -0,0 +1,17 @@
+# getting started
+
+## start gnoland chain
+gnoland-start
+
+## should show test1
+gnokey list
+
+## query test1
+gnokey query auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5
+
+# XXX: working but need an invitation
+stdin test1_password
```

Comment:
In the context of gnoland integrations, testing invalid passphrases may not be necessary; it seems more relevant for gnokey integration within gno.land/cmd/gnokey.

I recommend we streamline our process by eliminating the need to manage these passphrases.

One solution could be to refine the testscripts such that the map["gnokey"] definition automatically circumvents this requirement.

This is not blocking to merge the PR, just an optimization to make writing such tests more common over time.

Reply:
I want to get ride of the stdin command, but I didn't find a way to do it yet, and gnokey command don't have any command to let met do it without using stdin

Reply:
fixed using custom commands .IO